### ISSUE TITLE
lines messed up in Propertius

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios_nemo_ui#266]
from issue #193 
I did also notice that in https://texts-test.alpheios.net/text/urn:cts:latinLit:phi0620.phi001.perseus-lat3/passage/1.11, there is line 15 and below line 10. I checked on Scaife and I could see the problem there. the lines are messed up, basically in Scaife they go from 8 to 15, 10, 9, 11, 12, 13, 14 and 16. same in latin library and Phi latin text and Perseus. interesting that in Perseus there are two translations, the one from Kats does not have this problem, so I guess it's due to the translation of Mueller that everybody is using.

**Comment by balmas** [comment]
yeah, this is an issue with the underlying source text.  Should fix at some point but will not worry about it for the release.

**Comment by monzug** [comment]
I still see the issue in passage 1.11 (see screenshot) - also there are some repetition of text in https://texts.alpheios.net/text/urn:cts:latinLit:phi0620.phi001.alpheios-text-lat1/passage/1.2?subreference=1.2. that are not present in Scaife or Perseus
passage 1.1, lines 4,5,6: teque teque peregrinis vendere muneribus,
naturaeque naturaeque decus mercato perdere cultu,
nec nec sinere in propriis membra nitere bonis? 
line 19: nec nec Phrygium falso traxit candore maritum

![Screen Shot 2020-06-25 at 2 10 43 PM](https://user-images.githubusercontent.com/41396793/85717320-af228180-b6ed-11ea-86ea-f4ff47cd19c3.png)

**Comment by monzug** [comment]
there are few repetition of words in the Alpheios version of Propertius. (e.g. neque neque, teque teque, naturaeque naturaeque, nec nec etc)
